src/utils/formatPageInfo/index.ts

```diff
@@ -5,13 +5,55 @@ import { formatDBTimestampToObject } from "./timestampFormatter";
 import type { SignatureData, Witness, SignatureStatus } from "../../types";
 import * as nameResolver from "../../name_resolver";
 
+type MerkleProofStatus = "VALID" | "INVALID" | "DOMAIN_SNAPSHOT";
+type EtherscanResult = "true" | "false" | "Transaction hash not found" | string;
+
+// let errMsg
```

Comment:
I think we should instead put a permalink here: https://github.com/inblockio/aqua-verifier-js/blob/de3432e7309c642159e756ddf7dc49e602ab660d/index.js#L346-L354.

Reply:
I think we should add some typing or other documentation that is more reliable than a permalink. If the lines between 346-354 are ever occupied by other code then the linking will be meaningless. Can we consider adding some typing to the aqua verifier that is then exported and imported here?

Reply:
That link is a permalink, which basically links to a specific state of the code of a specific commit. Compare that with https://github.com/inblockio/aqua-verifier-js/blob/main/index.js#L346-L353=, which is a dynamic link to the latest main.

Reply:
I haven't done anything to improve the aqua-verifier-js repo for a while due to the plan to use aqua-verifier-go via wasm instead.

Reply:
thx for clarification :) I've updated docs with permalinks